src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,22 @@
+package bit.javaoop;
+
+import java.util.List;
+
+public abstract class Manager extends AbstractEmployee {
+    List<Employee> listofEmployees;
+
+    Manager(String name, double salary) {
+        super(name, salary);
+
+    }
+
+    public abstract boolean CanHire(Employee employee);
+    public void HireEmployee(Employee employee) {if(CanHire(employee)) listofEmployees.add(employee);}
```

Comment:
- Unikamy takich onelinerów w sensie ctrl + alt + L je załatwi automatycznie :D 
- Metoda ma złą nazwę nazwa metody powinna mówić o tym co robi dana metoda a spójrzmy co robi ta metoda
sprawdza czy da się zatrudnić pracownika jeśli tak to go zatrudnia
a więc HireEmployeeIfPossible byłoby zdecydowanie lepszą nazwą:)